fenix/app/build.gradle

```diff
@@ -488,6 +488,7 @@ dependencies {
     implementation FenixDependencies.androidx_constraintlayout
     implementation FenixDependencies.androidx_coordinatorlayout
     implementation FenixDependencies.google_accompanist_drawablepainter
+    implementation FenixDependencies.google_accompanist_pager_indicators
```

Comment:
what's this used for?

Reply:
Was originally using the accompanist pager indicators since they weren't part of the pager, but opted to use the pager indicators we added that was based on the the accompanist version.